samples/bluetooth/hci_uart/nrf52840_pca10090.overlay

```diff
@@ -2,7 +2,7 @@
 
 / {
 	chosen {
-		zephyr,bt-uart=&uart1;
+		zephyr,bt-c2h-uart=&uart1;
```

Comment:
Could this be added to the boards/arm/nrf52840_pca10090?

Reply:
No, because that board is highly configurable and we cannot assume a UART. By default the wires are routed to some pins instead.

Reply:
OK